cgi-bin/LJ/Web.pm

```diff
@@ -4569,7 +4572,7 @@ sub statusvis_message_js {
     $statusvis_full = "memorial" if $u->is_memorial;
     $statusvis_full = "readonly" if $u->is_readonly;
 
-    LJ::need_res("js/statusvis_message.js");
+    LJ::need_res( { group => 'all' }, "js/statusvis_message.js" );
```

Comment:
Wait, I just realized I should call this out for special attention — right now I believe this memorial/readonly status message thing is actually broken on prod, because it gets added to the `default` (read: "legacy") resource group and there's no way for an S2 page to actually be in that group; they're locked to `jquery`. Soooo this would fix it, but it only just now occurred to me to wonder if this is another of those The Ring "you weren't supposed to _help_ her" situations.

Reply:
AFAIK no one had realized it was broken. Memorial journals are rare (we get one or two requests for this per year) and we don't use readonly for any situations that I'm aware of.

Reply:
Wait.... actually, it's still broken even if I reenable this. That JS file calls `LiveJournal.register_hook()` instead of `$(document).ready()`, and the `livejournal.js` file that declares that function is never loaded in the jquery or foundation resource groups. 

Easy enough to un-break it for real, now that I understand what's happening. Do we wanna?

Reply:
I'm ambivalent about fixing this. As long as the profile can show memorial status, which should be the case, I'm OK if a journal can't know? Readonly we don't use.

That said it _attempts_ to do it, so I'd prefer we either remove the attempting or we fix it. Better to be clear one way or another than just broken.